olive/passes/pytorch/gptq.py

```diff
@@ -215,16 +197,17 @@ def get_onnx_quant_linear(*args, **kwargs):
         quantized_model = quantized_model.model
         assert self.accelerator_spec.accelerator_type == Device.GPU
 
-        converted_onnx_model = OnnxConversion._export_pytorch_model(  # pylint: disable=protected-access
-            quantized_model,
-            model.get_dummy_inputs(),
-            model.get_io_config(),
-            config,
-            config["export_device"],
-            None,
-            tempfile.tempdir,
+        output_model_path = Path(output_model_path).with_suffix(".pt")
```

Comment:
will the output_model_path be a directory or file name?

Reply:
@jambayk, I am suppose the output_model_path is a directory? do we need append model file name?

Reply:
Maybe output_model_path / "model.pt" is better?

Reply:
`output_model_path` is neither a file or directory https://github.com/microsoft/Olive/blob/d5c489ed60c91ea3ea9c1a28a90c2fe400e86bb1/olive/engine/engine.py#L915 

So you can either just attach a `.pt` suffix and save to it directly or do something like https://github.com/microsoft/Olive/blob/d5c489ed60c91ea3ea9c1a28a90c2fe400e86bb1/olive/passes/qnn/conversion.py#L108